kaggle_environments/agent.py

```diff
@@ -40,23 +76,19 @@ def url_agent(o, c, r, i):
                     **c,
                     "agentExec": "LOCAL"
                 },
-                "environment": environment,
+                "environment": environment.name,
```

Comment:
note that the environment passed to this method used to just be the environment name but is now the complete environment class so that we can resolve default agents from the environment as well.